### PR TITLE
Add icon for public transport shelter

### PR DESCRIPTION
**Pull request by dvdmrtnz**
* This PR adds icon for public transport shelter by @patepelo from https://github.com/organicmaps/organicmaps/issues/2319#issuecomment-2067806780

---

### CODE REVIEW

**Comment by patepelo** [comment]
Thank you @dvdmrtnz. One important thing, this is a redesign of an icon from Roentgen. We wanted to start including them and some modifications like this one. For this PR we would have to include the credits. Here is the origin: https://github.com/enzet/Roentgen and maybe consider adding credits for this other source which we would also like to start using:[ GitHub:sjjb.co.uk](https://github.com/twain47/Open-SVG-Map-Icons/tree/master/svg)

**Comment by dvdmrtnz** [comment]
Added Röntgen copyright notification

**Review by pastk — COMMENTED** [review]
Please change in the outdoors style also!

**Comment by dvdmrtnz** [comment]
> Please change in the outdoors style also!

Done

File: data/drules_proto_outdoors_clear.txt [hunk 1]
```diff
@@ -12395,7 +12395,7 @@ cont {
   element {
     scale: 13
     symbol {
-      name: "shelter-s"
+      name: "shelter-public_transport-m"
```
Comment (pastk) [inline data/drules_proto_outdoors_clear.txt:12398]: either we need an `-s` icon for z13,
or (better) let's make PT shelters  z14- in outdoors style (you'll have to reset the default icon on z13)

Comment (dvdmrtnz) [reply data/drules_proto_outdoors_clear.txt:12398]: I've removed it from z13

File: data/styles/outdoors/include/Icons.mapcss [hunk 2]
```diff
@@ -266,6 +266,10 @@ area|z16-[amenity=shelter],
 node|z16-[amenity=shelter][shelter_type=lean_to],
 node|z16-[amenity=shelter][shelter_type=basic_hut],
 {font-size: 12; icon-min-distance: 0;}
+node|z13-[amenity=shelter][shelter_type=public_transport],
+{icon-image: zero-icon.svg;}
```
Comment (pastk) [inline data/styles/outdoors/include/Icons.mapcss:270]: please avoid using a zero-icon hack (e.g. its user-selectable still)
please use `icon-image: none;` instead

Comment (pastk) [reply data/styles/outdoors/include/Icons.mapcss:270]: and `node|z13[amenity=...` is enough as we need to reset it for z13 only

Comment (dvdmrtnz) [reply data/styles/outdoors/include/Icons.mapcss]: Thanks. I didn't know that you could set the icon-image to none

File: data/styles/outdoors/include/priorities_4_overlays.prio.txt [hunk 3]
```diff
@@ -978,7 +978,7 @@ tourism-artwork-painting                            # icon z15- (also has captio
 === 1550
 
 amenity-shelter                                     # icon z13- (also has caption(optional) z13-)
-amenity-shelter-public_transport                    # icon z13- (also has caption(optional) z13-)
+amenity-shelter-public_transport                    # icon z14- and caption z13 (also has caption(optional) z14-)
```
Comment (pastk) [inline data/styles/outdoors/include/priorities_4_overlays.prio.txt:981]: also need to add `text: none;` to remove the z13 caption..

File: data/styles/outdoors/include/Icons.mapcss (hunk 2 above)
Comment (pastk) [reply data/styles/outdoors/include/Icons.mapcss]: yeap its a relatively new improvement :)
need to disable the caption also..

File: data/styles/outdoors/include/priorities_4_overlays.prio.txt (hunk 3 above)
Comment (dvdmrtnz) [reply data/styles/outdoors/include/priorities_4_overlays.prio.txt]: Okay! Should be good now!

**Review by pastk — APPROVED** [review]
LGTM
And thanks for your patience!

**Review by biodranik — APPROVED** [review]
Thanks! Wild shelters are the same, right?

**Comment by dvdmrtnz** [comment]
> Thanks! Wild shelters are the same as in the master now, right?

Yes. This only changes shelters tagged with `shelter_type=public_transport`.

**Pull request merged**